docs/clustering.md

```diff
@@ -249,7 +249,6 @@ There is nothing can be done.
 - Stop replication of errant replicas.
 - Set `super_read_only=1` for replica instances that are writable.
 - Adjust `moco.cybozu.com/role` label to Pods according to their roles.
-    - For errant replicas, the label is removed to prevent users from reading inconsistent data.
```

Comment:
Why do you remove this specification?

Reply:
I thought it would be easier to determine which instance is in progress changing roles.
Should I keep this specification?

Reply:
reverted 190f834fa99421daece8c99ac110099d496eb666

Reply:
Actually, preventing users from accessing errant replicas is more important IMO.

Reply:
ah, I understood your intention.
You wanted to add `errant-replica` as a role to indicate that the instance is neither primary nor a replica.

Reply:
> For errant replicas, the label is removed to prevent users from reading inconsistent data.

How about adding this line instead?

```
- For errant replicas, the label is set to `errant-replica`, which effectively prevents clients from connecting them via a Service.
```

Reply:
@ymmt2005 
Thanks for the advice.
Yes. I was trying to add `moco.cybozu.com/role: errant-replica` label to errant replicas and to kill the connections on the errants just in case.

However, I think killing the errant replica's connections is almost ineffective.
And I could fix the issue https://github.com/cybozu-go/moco/issues/597 without the breaking change.
So, I will not change the specification of the role label.

Reply:
OK.